src/components/icons/base/Icon.svelte

```diff
@@ -89,6 +89,8 @@
       "ms-transform": `rotate(${rotate}deg)`,
       transform: `rotate(${rotate}deg)`
     };
+  } else {
```

Comment:
Haaa good catch, this was not reactive right?

Reply:
Yes I believe so. This way it'll rotate back if it's set to `null`